src/map.react.js

```diff
@@ -303,6 +305,20 @@ export default class MapGL extends Component {
     map.addSource(update.id, newSource);
   }
 
+  // Hover and click only query layers whose interactive property is true
```

Comment:
Mapbox's documentation for layer.interactive: 
https://www.mapbox.com/mapbox-gl-style-spec/#layer-interactive